CONTRIBUTING.adoc

```diff
@@ -50,15 +50,23 @@ If the project is _not_ intended to be used for production:
 * *MUST* clearly labeled in your repos README.adoc as `NOT INTENDED FOR PRODUCTION USE`
 
 
+=== Code Reviews
+Code reviews *MUST* be performed by at least one committer before a pull request can be merged.
 
-=== Code Contributions
-Contribution pull requests *MUST* contain the following:
+Reviewers *SHOULD* make suggestions in lieu of just comments, for issues found in pull request following github's guidelines https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/reviewing-proposed-changes-in-a-pull-request[here]
```

Comment:
... found in a pull request following https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/reviewing-proposed-changes-in-a-pull-request[GitHub's guidelines].

Reply:
fixed, thanks